src/components/periodic-table/PeriodicTable.tsx

```diff
@@ -161,6 +161,7 @@ class PeriodicTable extends React.PureComponent<
               <div
                 className="periodic-table__cell"
                 key={`row-${i}-cell-${index}`}
+                aria-label={`atomic-number-${element}`}
```

Comment:
Having `atomic-number` as the `aria-label` seems weird, keep in mind that this is that a screen reader will read, imagine it like an inputs label, you wouldn't use `company-email` as the label but `Company email`.

I'm not sure if in this case we should go for an `aria-label`, I don't think it's really useful for the user since the `<$component />` will identify the element, there is no need for `aria-label`, what do you think about using a `data-testid` here?

cc @mpeyper @tigerabrodi

Reply:
@rubenmoya if data-testid is really needed for the test, then sure.
Don't forget, we got Testing Playground :wink:
IMO Test ids should be the last resort.

My suggestion would be to remove the aria label here, and we can still make the website more accessible later :raised_hands: